app/views/state_file/questions/md_review/edit.html.erb

```diff
@@ -123,8 +131,7 @@
 
       <div class="spacing-below-25">
         <p class="text--bold spacing-below-5"><%= t("state_file.general.md_nonrefundable_local_earned_income_tax_credit") %></p>
-        <!--        <p><%#= number_to_currency(current_intake.calculator.line_or_zero(:MD502_LINE_29)) %></p>-->
-        WIP
+        <p><%= number_to_currency(current_intake.calculator.line_or_zero(:MD502_LINE_29)) %></p>
       </div>
```

Comment:
[pebble] theres a todo for this in the md review spec (spec/controllers/state_file/questions/md_review_controller_spec.rb) to add the tests for these lines that you shows now

Reply:
hmm i'm seeing that spec spec running fully (not sure how it's passed before now)